dashboard/src/components/resource-list/ResourceListView.js

```diff
@@ -99,7 +99,7 @@ export const ResourceListView = ({
       },
       { title: "Revision", field: "revision" },
       {
-        title: "Version",
+        title: "Variants",
         field: "versions",
```

Comment:
should this still be versions?

Reply:
it displays variants but retrieves the portions of the json file named "version" for logical purposes. Quicker to change this now than change the name for everything, though will fix that later.